src/main/java/net/dv8tion/jda/internal/requests/restaction/MessageActionImpl.java

```diff
@@ -258,6 +263,22 @@ public MessageActionImpl override(final boolean bool)
         return this;
     }
 
+    private String applyOptions(String name, AttachmentOption[] options)
+    {
+        if (options.length > 0)
```

Comment:
We don't need to check if the options has any length. The loop will just skip if none were provided.